diff -ur a/src/libsodium/crypto_secretbox/xsalsa20poly1305/secretbox_xsalsa20poly1305.c b/src/libsodium/crypto_secretbox/xsalsa20poly1305/secretbox_xsalsa20poly1305.c
--- src/libsodium/crypto_secretbox/xsalsa20poly1305/secretbox_xsalsa20poly1305.c	2019-05-30 13:13:18.000000000 +0000
+++ src/libsodium/crypto_secretbox/xsalsa20poly1305/secretbox_xsalsa20poly1305.c	2022-01-13 13:42:44.891332633 +0000
@@ -28,7 +28,7 @@
                                        const unsigned char *n,
                                        const unsigned char *k)
 {
-    unsigned char subkey[32];
+    unsigned char subkey[32] = {0};
     int           i;

     if (clen < 32) {
