diff -ru src/backup.c src/backup.c
--- src/backup.c	2020-03-29 22:57:07.916530752 +0000
+++ src/backup.c	2020-03-29 23:04:44.671567956 +0000
@@ -12,8 +12,7 @@
 ** This file contains the implementation of the sqlite3_backup_XXX() 
 ** API functions and the related features.
 */
-#include "sqliteInt.h"
-#include "btreeInt.h"
+#include "crypto.h"
 
 /*
 ** Structure allocated for each backup operation.
diff -ru src/crypto.c src/crypto.c
--- src/crypto.c	2020-03-29 23:07:54.203424377 +0000
+++ src/crypto.c	2020-03-29 23:07:50.403428027 +0000
@@ -32,7 +32,6 @@
 #ifdef SQLITE_HAS_CODEC
 
 #include <assert.h>
-#include "sqlcipher.h"
 #include "crypto.h"
 
 #ifdef SQLCIPHER_EXT
diff -ru src/crypto.h src/crypto.h
--- src/crypto.h	2020-03-29 23:05:01.111558972 +0000
+++ src/crypto.h	2020-03-29 23:04:55.803561954 +0000
@@ -37,6 +37,7 @@
 
 #include "sqliteInt.h"
 #include "btreeInt.h"
+#include "sqlcipher.h"
 #include "pager.h"
 
 /* extensions defined in pager.c */ 
diff -ru src/crypto_impl.c src/crypto_impl.c
--- src/crypto_impl.c	2020-03-29 23:08:31.451387166 +0000
+++ src/crypto_impl.c	2020-03-29 23:08:27.563391170 +0000
@@ -31,7 +31,6 @@
 /* BEGIN SQLCIPHER */
 #ifdef SQLITE_HAS_CODEC
 
-#include "sqlcipher.h"
 #include "crypto.h"
 #ifndef OMIT_MEMLOCK
 #if defined(__unix__) || defined(__APPLE__) || defined(_AIX)
